recipes/default.rb

```diff
@@ -17,6 +17,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# both are used if used with Chef Compliance directly, otherwise those should be nil
```

Comment:
`used if used` jumped up here. Alternative:
`# These two attributes should only be set when connecting directly to Chef Compliance, otherwise they should be nil`